Source/ParameterEncoding.swift

```diff
@@ -261,6 +293,24 @@ public struct URLEncoding: ParameterEncoding {
             return false
         }
     }
+
+    private func encodeArrayKey(_ key: String) -> String {
+        switch arrayEncoding {
+        case .brackets:
+            return "\(key)[]"
+        case .noBrackets:
+            return key
+        }
+    }
+
+    private func encodeBoolValue(_ value: Bool) -> String {
```

Comment:
Like the array encoding function, this should be an `encode` function on `BoolEncoding`.